### DISCUSSION TITLE
How to get net generation of a service area by fuel type?

### DISCUSSION THREAD

**Discussion by bendnorman** [catalyst-cooperative/pudl#2287] [Help Me!]
I saw this [cool chart](https://twitter.com/joshdr83/status/1615406893836472340) from @joshdr83 on Twitter that has ERCOT's fuel mix since 2006. I'd like to recreate this chart for Alaska's rail belt utilities or really any service area/collection of utilities using PUDL data. What is the best way to do this?

I managed to create a chart that roughly aligned with [EIA's reported fuel mix](https://www.eia.gov/tools/faqs/faq.php?id=427&t=3) for the US by concatenating the `generation_fuel_nuclear_eia923` and `generation_fuel_eia923` tables then merging them with `plants_entity_eia` to get the generator's state. I then grouped generators by `fuel_type_code_pudl` and summed the `net_generation_mwh` column.

Should I be using the `gf_eia923` output table instead?

**Comment by grgmiller** [comment]
Check out the Open Grid Emissions Dataset that we built using pudl inputs: https://singularity.energy/open-grid-emissions. Specifically, the power sector data will give you fuel mix at the annual, monthly, or hourly resolutions. Currently we only have data for 2019-2021, but we're working on expanding the historical coverage.
